plugins/user-authenticators/saml2/src/main/java/org/apache/cloudstack/saml/SAMLUtils.java

```diff
@@ -100,8 +100,11 @@
 public class SAMLUtils {
     public static final Logger s_logger = Logger.getLogger(SAMLUtils.class);
 
+    static final String charset = "abcdefghijklmnopqrstuvwxyz";
```

Comment:
should we also have A-Z (i.e. a-z in upper case too) @weizhouapache ?

Reply:
@rohityadavcloud 
I have checked hundreds of request IDs, all are composed by digits (0 to 9) or lower case alphabet (a to z).